libraries/Servo/src/Servo.h

```diff
@@ -63,8 +63,15 @@ class Servo
 public:
     Servo();
     ~Servo();
-    uint8_t attach(int pin);           // attach the given pin to the next free channel, sets pinMode, returns channel number or 0 if failure
-    uint8_t attach(int pin, uint16_t min, uint16_t max); // as above but also sets min and max values for writes. 
+    // attach the given pin to the next free channel, sets pinMode, returns channel number or 0 if failure.
+    uint8_t attach(int pin);
+    // attach the given pin to the next free channel, sets pinMode, min, and max values for writes.
+    // returns channel number or 0 if failure.
+    uint8_t attach(int pin, uint16_t min, uint16_t max);
+    // attach the given pin to the next free channel, sets pinMode, min, max values for writes,
+    // and sets the initial value, the same as write.
+    // returns channel number or 0 if failure.
+    uint8_t attach(int pin, uint16_t min, uint16_t max, int value);
```

Comment:
@Tech-TX Have you tried either of these two overloads of `attach()`?

Reply:
I pulled it this morning, Dirk, but I missed the attach() parameters.  :wink:  

I *really* need to add a bunch of *.rst or *.md help files or additions for the stuff in the libraries, especially where things differ from the original Arduino libraries.  That would help minimize confusion for our Gentle Users (and me!).  The current info at ReadTheDocs is woefully incomplete for the servo library.  Here's the full quote:

>**Servo**
This library exposes the ability to control RC (hobby) servo motors. It will support up to 24 servos on any available output pin. By default the first 12 servos will use Timer0 and currently this will not interfere with any other support. Servo counts above 12 will use Timer1 and features that use it will be affected. While many RC servo motors will accept the 3.3V IO data pin from a ESP8266, most will not be able to run off 3.3v and will require another power source that matches their specifications. Make sure to connect the grounds between the ESP8266 and the servo motor power supply.

I didn't know we had 24 I/O pins that could drive servos.  You learn something every day...  and now I need to hunt down the 13 other (missing) GPIO pins. :smile: 

Here's the info on the Arduino library:  https://www.arduino.cc/en/Reference/ServoAttach
so the 8266 library has an INT VALUE option for the initial starting degrees that the Arduino library doesn't.

Your PR is working OK with my test bed and servos.